go.mod

```diff
@@ -3,12 +3,7 @@ module github.com/hashicorp/vault-plugin-auth-azure
 go 1.19
 
 require (
```

Comment:
I think needs a `go mod tidy` before we submit this?

Reply:
Good catch, fixed!